src/neynar-api/v2/openapi-farcaster/apis/user-api.ts

```diff
@@ -592,6 +594,43 @@ export const UserApiAxiosParamCreator = function (configuration?: Configuration)
 
 
     
+            setSearchParams(localVarUrlObj, localVarQueryParameter);
+            let headersFromBaseOptions = baseOptions && baseOptions.headers ? baseOptions.headers : {};
+            localVarRequestOptions.headers = {...localVarHeaderParameter, ...headersFromBaseOptions, ...options.headers};
+
+            return {
+                url: toPathString(localVarUrlObj),
+                options: localVarRequestOptions,
+            };
+        },
+        /**
+         * Fetches power users and respond in a backwards compatible format to Warpcast\'s deprecated power badge endpoint.
+         * @summary Fetch power user FIDs
+         * @param {string} apiKey API key required for authentication.
+         * @param {*} [options] Override http request option.
+         * @throws {RequiredError}
+         */
+        userPowerLite: async (apiKey: string, options: AxiosRequestConfig = {}): Promise<RequestArgs> => {
```

Comment:
Need wrapper code for `userPowerLite`

Reply:
I'll add this wrapper code in a separate PR on Monday 👍